cross/syncthing/PLIST

```diff
@@ -1 +1,2 @@
 bin:bin/syncthing
+rsc:gui/default/
```

Comment:
```suggestion
rsc:gui/default/tech-ui/
```

Probably doesn't change anything, but this could be misleading to think that the regular GUI will also be installed as source, while it is really only bundled in the binary.